ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/SourceBuilder.java

```diff
@@ -212,44 +211,31 @@ public static SourceBuilder instance() {
     );
   }
 
-  private static PhysicalSchema getPhysicalSchemaWithKeyAndPseudoCols(
+  private static PhysicalSchema getPhysicalSchemaWithRelevantPseudoAndKeyCols(
```

Comment:
I don't understand these changes. How come the two previous methods `getPhysicalSchemaWithKeyAndPseudoCols()` and `getPhysicalSchemaWithPseudoColumnsToMaterialize()` were collapsed into a single implementation? Does the functionality still work with these changes?

Reply:
It's still functional because we always use the new functionality with `TableSource{V2}`s and the old functionality (adding pseudo and key cols) with `WindowedTableSource{V2}`.

Reply:
OK, I probably misread an earlier version of this PR. Can we rename this method to `getPhysicalSchemaWithPseudoColumnsToMaterialize()` in order to clarify the method's behavior? "Relevant pseudo and key columns" isn't very informative as a name.